lib/spack/docs/environments.rst

```diff
@@ -9,139 +9,279 @@
-interaction will occur through the ``spack.yaml`` file and the Spack
-commands that affect it. When the environment is concretized, Spack
-will create a file ``spack.lock`` with the concrete information for
-the environment.
+interactions will occur through the ``spack.yaml`` file and the
+Spack commands that affect it. When the environment is concretized,
+Spack will create a file ``spack.lock`` with the concrete information
+for the environment.
 
-In addition to being the default location for the view associated with
-an Environment, the ``.spack-env`` directory also contains:
+In addition to being the default location for the view associated
+with an Environment, the ``.spack-env`` directory also contains
+the following subdirectories:
 
-  * ``repo/``: A repo consisting of the Spack packages used in this
+  * ``repo``: A repo consisting of the Spack packages used in this
     environment.  This allows the environment to build the same, in
     theory, even on different versions of Spack with different
     packages!
-  * ``logs/``: A directory containing the build logs for the packages
+  * ``logs``: A directory containing the build logs for the packages
     in this Environment.
 
-Spack Environments can also be created from either a ``spack.yaml``
-manifest or a ``spack.lock`` lockfile. To create an Environment from a
-``spack.yaml`` manifest:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Creating an Independent Environment
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+As mentioned previously, independent environments are those
+whose environment files do not reside within Spack's managed
+environment directory. In other words, any directory can be
+treated as an environment.
+
+You can create a new independent environment in the current
+directory using:
+
+.. code-block:: console
+
+   $ spack env create -d .
+
+In this case Spack creates a ``spack.yaml`` file in the present
+directory.
+
+Alternatively, you can create an independent environment from an
+existing ``spack.yaml`` manifest or a ``spack.lock`` lockfile.
+In either case you can work in the directory where the files
+reside or pass the full path to the files.
+
+To create ``myenv`` from a ``spack.yaml`` manifest in the current
+directory, enter:
 
 .. code-block:: console
 
    $ spack env create myenv spack.yaml
 
-To create an Environment from a ``spack.lock`` lockfile:
+.. note::
+
+   A Spack environment created from a ``spack.yaml`` manifest
+   is guaranteed to have the same root specs as the original
+   environment, but may concretize differently.
+
+If you want a more exact reproduction of the existing environment,
+you can use the ``spack.lock`` lockfile by entering:
 
 .. code-block:: console
 
    $ spack env create myenv spack.lock
 
-Either of these commands can also take a full path to the
-initialization file.
+.. note::
+
+   A Spack environment created from a ``spack.lock`` lockfile
+   is guaranteed to have the same concrete specs as the original
+   environment.
```

Comment:
```suggestion
   A Spack environment created from a ``spack.lock`` lockfile
   is guaranteed to have the same abstract and concrete specs as the original
   environment, but it will not preserve configuration options.
```